rv32c.yml

```diff
@@ -0,0 +1,30 @@
+# ISA Configurations
```

Comment:
Remove this file please -- but update sparce.yml and example.yml to contain the new "rv32c_enabled" option. Set it to "disabled" by default.